src/main/scala/org/broadinstitute/dsde/firecloud/service/StatusService.scala

```diff
@@ -1,43 +1,62 @@
 package org.broadinstitute.dsde.firecloud.service
 
-import java.text.SimpleDateFormat
-import java.util.Date
-
 import akka.actor.Actor
-import spray.json._
+import akka.actor.Props
+import akka.pattern._
+import org.broadinstitute.dsde.firecloud.Application
+import org.broadinstitute.dsde.firecloud.dataaccess.{AgoraDAO, RawlsDAO, SearchDAO, ThurloeDAO}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impSystemStatus
+import org.broadinstitute.dsde.firecloud.model.{SubsystemStatus, SystemStatus}
+import org.broadinstitute.dsde.firecloud.service.PerRequest.{PerRequestMessage, RequestComplete}
+import org.broadinstitute.dsde.firecloud.service.StatusService.CollectStatusInfo
+import spray.http.StatusCodes
+import spray.httpx.SprayJsonSupport
+import spray.json.DefaultJsonProtocol._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
+
+/**
+ * Created by anichols on 4/5/17.
+ */
+object StatusService {
+  def props(statusServiceConstructor: () => StatusService): Props = Props(statusServiceConstructor())
+
+  def constructor(app: Application)()(implicit executionContext: ExecutionContext): StatusService = {
+    new StatusService(app.searchDAO, app.agoraDAO, app.thurloeDAO, app.rawlsDAO)
```

Comment:
optional feedback: at this point, you might just want to pass the entire `app` object, instead of extracting parts.

In looking at the `Application` object, I realize we need additional status checks! Those are for a future story.